pkg/internal/infrastructure/terraform.go

```diff
@@ -52,7 +52,7 @@ const (
 	MaxApiCallRetries = "10"
 )
 
-// StatusTypeMeta is the TypeMeta of the GCP InfrastructureStatus
+// StatusTypeMeta is the TypeMeta of the InfrastructureStatus
 var StatusTypeMeta = metav1.TypeMeta{
 	APIVersion: apiv1alpha1.SchemeGroupVersion.String(),
 	Kind:       "InfrastructureStatus",
```

Comment:
I keep seeing the kind hardcoded in the code - is there no const available somewhere?
Just wondering, I don't see us actually changing the Kind anytime soon.